plugins/stk/mallets/mallets.cpp

```diff
@@ -506,7 +519,9 @@ malletsSynth::malletsSynth(const StkFloat _pitch, const StkFloat _velocity, cons
 		m_voice->controlChange(128, 128.0f);
 
 		m_voice->noteOn(_pitch, _velocity);
-	} catch (...) { m_voice = nullptr; }
+	} catch (...) {
```

Comment:
Same here.

Reply:
You mean like

```
try
{
    // ...
}
catch
{
    // ...
}
```
?

And the same for if-else?

Reply:
Exactly.

Reply:
Should be fixed now.